spec/Section 4 -- Introspection.md

```diff
@@ -119,106 +119,10 @@ warnings.
 
 **Schema Introspection Schema**
 
-The schema introspection system is itself represented as a GraphQL schema. Below
-are the full set of type system definitions providing schema introspection,
-which are fully defined in the sections below.
+The schema introspection system is itself represented as a GraphQL schema.
 
-```graphql
-type __Schema {
-  description: String
-  types: [__Type!]!
-  queryType: __Type!
-  mutationType: __Type
-  subscriptionType: __Type
-  directives: [__Directive!]!
-}
-
-type __Type {
-  kind: __TypeKind!
-  name: String
-  description: String
-  # must be non-null for OBJECT and INTERFACE, otherwise null.
-  fields(includeDeprecated: Boolean = false): [__Field!]
-  # must be non-null for OBJECT and INTERFACE, otherwise null.
-  interfaces: [__Type!]
-  # must be non-null for INTERFACE and UNION, otherwise null.
-  possibleTypes: [__Type!]
-  # must be non-null for ENUM, otherwise null.
-  enumValues(includeDeprecated: Boolean = false): [__EnumValue!]
-  # must be non-null for INPUT_OBJECT, otherwise null.
-  inputFields(includeDeprecated: Boolean = false): [__InputValue!]
-  # must be non-null for NON_NULL and LIST, otherwise null.
-  ofType: __Type
-  # may be non-null for custom SCALAR, otherwise null.
-  specifiedByURL: String
-}
-
-enum __TypeKind {
-  SCALAR
-  OBJECT
-  INTERFACE
-  UNION
-  ENUM
-  INPUT_OBJECT
-  LIST
-  NON_NULL
-}
-
-type __Field {
-  name: String!
-  description: String
-  args(includeDeprecated: Boolean = false): [__InputValue!]!
-  type: __Type!
-  isDeprecated: Boolean!
-  deprecationReason: String
-}
-
-type __InputValue {
-  name: String!
-  description: String
-  type: __Type!
-  defaultValue: String
-  isDeprecated: Boolean!
-  deprecationReason: String
-}
-
-type __EnumValue {
-  name: String!
-  description: String
-  isDeprecated: Boolean!
-  deprecationReason: String
-}
-
-type __Directive {
-  name: String!
-  description: String
-  locations: [__DirectiveLocation!]!
-  args(includeDeprecated: Boolean = false): [__InputValue!]!
-  isRepeatable: Boolean!
-}
-
-enum __DirectiveLocation {
-  QUERY
-  MUTATION
-  SUBSCRIPTION
-  FIELD
-  FRAGMENT_DEFINITION
-  FRAGMENT_SPREAD
-  INLINE_FRAGMENT
-  VARIABLE_DEFINITION
-  SCHEMA
-  SCALAR
-  OBJECT
-  FIELD_DEFINITION
-  ARGUMENT_DEFINITION
-  INTERFACE
-  UNION
-  ENUM
-  ENUM_VALUE
-  INPUT_OBJECT
-  INPUT_FIELD_DEFINITION
-}
-```
+See [Appendix C](#sec-Introspection-types) for more details about the schema
+type definitions.
```

Comment:
This should not be removed; it's heavily referenced whilst reading this section of the spec. It's okay to _repeat_ it in an appendix, like how we repeat the grammar in the appendix.

Reply:
Good callout 👍